specs/darwin/signature.table

```diff
@@ -6,6 +6,9 @@ schema([
     Column("hash_resources", INTEGER,
        "Set to 1 to also hash resources, or 0 otherwise. Default is 1",
        additional=True),
+    Column("hash_executable", INTEGER,
```

Comment:
Do you think there is any case where the user might want one, but not the other? 

Because I wonder if `hash_resources` should cover `kSecCSDoNotValidateExecutable`. I am not honestly sure.

Reply:
I'm wondering if someone `hash_resources = 0` do they expect the executable to be validated? And is it meaningful to validate the executable but not the resources?

I can go along with either way. Though I tend to think fewer options is less confusing.

Reply:
Agree, I'm all ears here.
Making `hash_resources = 0` not hash resources & executable makes sense to me too.
It is a bit of a breaking change (of table behavior) though, but if we are ok with this then we can just document this on the release notes.

Reply:
@zwass

Reply:
Chatting in office hours, there's some support for mirror apple's SDK, and leaving these split.